dashboard/client/public/index.html

```diff
@@ -11,7 +11,7 @@
   <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico?v=1">
   <title>OpenFaaS Cloud</title>
   <!-- inject global variables from the server side -->
-  <script>
+  <script id="server_data" data-signed-in="__IS_SIGNED_IN__">
```

Comment:
I would suggest adding a span or *something else* to hold the data attribute.

Reply:
As we discussed, I used approach already present and I placed it inside script as a global variable